config/default/editor.editor.filtered_html.yml

```
@@ -116,7 +118,7 @@ settings:
         - '<div class>'
         - '<blockquote cite class>'
         - '<ul type class>'
-        - '<ol start type reversed class>'
+        - '<ol type class>'
```

```suggestion
        - '<ol start type reversed class>'
```
we don't want to lose this

---

Meant to make a note of this. That happened automatically with the update + re-export. Not sure this will survive another export.

---

yeah, because it is so touchy, i think we should keep this thread unresolved until we are ready to merge.